lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -772,6 +772,12 @@ def cmd_services(*args)
         matched_service_ids << service.id
 
         if mode == :update
+          if service.name == "unknown"
+            arr = []
+            arr << service.id 
+            framework.db.delete_service(ids: arr)
+            next 
+          end 
```

Comment:
I'm not sure that we want this since it does the same thing as `services --delete -s unknown` and makes it impossible to modify a service that is unknown. If we do keep this, I would added under a different option, like `--delete-unknown`?

Reply:
If the unknown service is being deleted from database , then is it required for renaming the "unknown" service ? since there would be no unknown service in the database after deletion .